docker/init.sh

```diff
@@ -7,8 +7,8 @@ make -s start-support
 node server.js --migrate-and-exit >/dev/null
 
 if [[ $NODEMON == "true" ]]; then
```

Comment:
I left the `NODEMON` environment variable alone for now. Any thoughts on better ways to handle this?

Reply:
What about adding support for a `DEV` environment variable and supporting both of them, but only describing `DEV` in the developer docs?

Reply:
I like it! I'll make that change.

Reply:
Done in de70d06 - want to give that a final look?

Reply:
Looks great!